test/food-route-test.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const expect = require('chai').expect;
+const request = require('superagent');
+const Food = require('../model/food.js');
+const url = 'http://localhost:8000';
```

Comment:
make sure that the port you pass here as your URL matches the port you explicitly create in your server.js file. When they aren't the same it can cause your test to error out because the server comes up on a different port than your test file is attempting to run on.